components/uniset/src/uniset.rs

```diff
@@ -341,12 +360,14 @@ impl UnicodeSet {
         if set.size() > self.size() {
             return false;
         }
-        let mut set_ranges: Chunks<u32> = set.as_inversion_list().chunks(2);
+        let inv_list = set.to_inversion_list();
```

Comment:
`to_inversion_list` allocates.  You should not allocate in a function like this!

Reply:
Done.